adamsjr8576/palette-picker-fe
# Testing forms

### What does this PR do?
- Adds react DOM testing library
- Up dates syntax to match your incoming changes 
- Tests `ProjectForm` to 100%
- add files to ignore during testing to `package-json`

### How to test?
- pull down `testing-forms` branch
- you will need to npm install again to include library  
- run `npm test` 
- I added the `--verbose` so we can see our ✅

### Issues:
- This PR closes no issues 
- I have **not** updated the project board yet

### Notes
- With this library there a few difference that we could use or not use, for example `test` instead of `it` among other things. We should make those decisions in this PR thread so we can get rolling? I have included the link to the docs for the library [here](https://testing-library.com/docs/dom-testing-library/intro). Yet me know if you have any questions.

@adamsjr8576 
Thanks!
